gcp/live/dev/k8s/stackdriver/monitoring/terraform.tfvars

```diff
@@ -20,3 +20,4 @@ terragrunt = {
 # ↓ Module configuration (empty means all default)
 
 ssl_enabled_uptime_checks = false
+notification_email = "alerts+dev@raisingthefloor.org"
```

Comment:
If I'm reading this correctly, this value should never be used. If this is correct, can you make it clearer that if we are seeing this value in an alert email then something has gone wrong? `alerts+dev-this-value-should-have-been-overridden@` or something?

Reply:
Good idea.